CHANGELOG.md

```diff
@@ -1,21 +1,26 @@
 # Changelog
 
+## 2.0.0
+
+Switch Windows memory allocation to use `CoTaskMemAlloc` and `CoTaskMemFree`,
+which will enable support for `NativeFinalizer`.
+
```

Comment:
Nit: This release requires Dart `2.17.0` or greater.

Reply:
Great! Updated.